docs/slides/day4.md

```diff
@@ -0,0 +1,524 @@
+class: middle, center
+
+# JumpStart Live Day 4
+Classroom exercises
+
+---
+
+# Instructor code review
+
+From [day 3 exercises](https://github.com/Ada-Developers-Academy/jump-start-live/tree/master/lessons/day3#exercises)
+
+You don't trust your users. Modify the program below to
+require the user to enter the same value twice in order to add that value to
+the total.
+```ruby
+puts "Hello! We are going to total some numbers!"
+puts "Enter a negative number to quit."
+
+total = 0
+input = gets.chomp.to_i
+while input > -1
+  total += input
+  input = gets.chomp.to_i
+end
+
+puts "Result: #{total}"
+```
+
+---
+
+# Instructor code review
+
+```ruby
+puts "Hello! We are going to total some numbers!"
+puts "Each number should be entered twice to be accepted."
+puts "Enter a negative number (twice) to quit."
+
+total = 0
+input = gets.chomp.to_i
+confirmedInput = gets.chomp.to_i
+
+while true # infinite loop, quit on break
+  if input == confirmedInput
+    if input < 0 # time to quit
+      break
+        else # add to total
```

Comment:
Should this section be aligned with the `if`?

Reply:
Normally yes, but this is on a code review slide, so that's a purposeful mistake.